bluesky_browser/viewer/image.py

```diff
@@ -77,10 +81,37 @@ def subfactory(self, name, descriptor_doc):
         image_keys = {}
         for key, data_key in descriptor_doc['data_keys'].items():
             ndim = len(data_key['shape'] or [])
+            # We want to record a shape that will match the arr.shape
+            # of the arrays we will see later. Ophyd has been writing
+            # incorrect info into descriptors. We try to detect and corect
+            # that here.
             if ndim == 2:
                 image_keys[key] = data_key['shape']
             elif ndim == 3:
-                image_keys[key] = data_key['shape'][1:]
+                # ophyd <1.4.0 gives (x, y, z) where z is 0
+                # Maybe the better way to detect this is start['version']['ophyd'].
```

Comment:
Shouldn't it be `versions` as in https://github.com/bluesky/bluesky/blob/5d19bb1f77a9fd7062b248a33ac7f80136af464f/bluesky/run_engine.py#L237-L241?

Reply:
Yes. Fixed.

Reply:
Ups, I just checked and don't see it. Not a big deal as it's a comment for now, but if used directly, would be nice to have a valid field name.